bHY6yh-de2985/cod-bxxqqq_bHY6yh-de2985.xml

```diff
@@ -0,0 +1,195 @@
+   <teiHeader>
+      <fileDesc>
+         <titleStmt>
+            <title>Rubrica 11</title>
+            <author ref="#per-c7y65a">Geremia da Montagnone</author>
+            <respStmt>
+               <name ref="#jeffreycwitt">Jeffrey C. Witt</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 11</title>
+               <date when="2023-05-22">May 22, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+            <witness xml:id="B" n="cod-bxxqqq">Paris, Bibliothèque Nationale de France, Nouv. acq. lat. 2469</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-05-22" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#B" n="6-r"/>
+            <cb ed="#B" n="b"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-de2985"><!-- p1l1-r11 -->
+            <head xml:id="bHY6yh-de2985-Hd1e108">Rubrica 11</head>
+            <head xml:id="bHY6yh-de2985-Hd1e111" type="question-title">De verbo dei et exauditione eius</head>
+            <p xml:id="bHY6yh-de2985-d1e114">
+              De verbo dei et 
+              <lb ed="#B"/>exauditione eius. Rubrica. <sic>II</sic><!--there are unambigious tittles on them--> 
+    </p>
+            <p xml:id="bHY6yh-de2985-d1e121">
+              <lb ed="#B"/>
+              <ref xml:id="bHY6yh-de2985-Rd1e118">/ In leuitico capitulo 26 et</ref> <quote xml:id="bHY6yh-de2985-Qd1e125">penulti Si in
```

Comment:
I would put the / outside the `<ref>` in this case. The slash is definitely relevant punctuation, but it's not really part of the ref. 

So something like: 

```
<lb/> / <ref>In leuitico
```